services/kube-oidc-proxy/0.3.0/defaults/cm.yaml

```diff
@@ -10,7 +10,7 @@ data:
     ---
     image:
       repository: quay.io/jetstack/kube-oidc-proxy
-      tag: v0.2.0
+      tag: v0.3.0
```

Comment:
This section can probably be removed since it's set in the chart values. Not a huge deal but might be better in case we ever forget to bump the override here in a future upgrade.
https://github.com/mesosphere/charts/pull/1273/files#diff-a7ea8b8d13b461b398365a7cf1ef25318683c21d5103961c74fec896d0df8d0fR9

Reply:
Removed, please have another look and merge.